src/Library/Plugin/UpdateData.php

```diff
@@ -210,8 +212,12 @@ public function fetchResponseData( $force = false ) {
 					),
 				)
 			);
+			if ( isset( $plugin_information->no_update ) ) {
```

Comment:
Why does no_update indicate no unusable data?

Reply:
If a request is successful and contains the data pulled from the WordPress.org repo, it will not contain a no_update property. That property was present in data returned by plugins_api() requests that were affected by outside filters.